test/functional/ui/cmdline_spec.lua

```diff
@@ -120,43 +120,43 @@ local function test_cmdline(linegrid)
       screen:try_resize(50,10)
       feed(':echoerr doesnotexist<cr>')
       screen:expect{grid=[[
-                                                          |
+        ^                                                  |
+        {1:~                                                 }|
+        {1:~                                                 }|
         {1:~                                                 }|
         {1:~                                                 }|
         {1:~                                                 }|
         {1:~                                                 }|
         {1:~                                                 }|
-        {3:                                                  }|
+        {3:n                                                 }|
         {4:E121: Undefined variable: doesnotexist}            |
-        {4:E15: Invalid expression: doesnotexist}             |
-        {5:Press ENTER or type command to continue}^           |
       ]]}
```

Comment:
@bfredl are these tests still semantically valid? Looks like they don't "scroll" now.

Reply:
A quick and simple substitute is to change every `:echoerr doesnotexist<cr>` to `:echoerr "err1" | echoerr "err2"<cr>`, no need to adjust height or columns or anything.